client/src/utils/tone/index.ts

```diff
@@ -62,12 +62,19 @@ export async function playTone(state: ToneStates): Promise<void> {
   const playSound = !!store.get('fcc-sound');
   if (playSound && toneUrls[state]) {
     const tone = await import('tone');
+
     if (tone.context.state !== 'running') {
       tone.context.resume().catch(err => {
         console.error('Error resuming audio context', err);
       });
     }
     const player = new tone.Player(toneUrls[state]).toDestination();
+
+    const storedVolume = (store.get('soundVolume') as number) ?? 50;
+    const calculateDecibel = -60 * (1 - storedVolume / 100);
```

Comment:
This is a bit tricky the way the decibels are calculated.

-60 is really quiet and 0 is really hard. That's why the percentage needs to be inverted.